src/components/avatars/OcAvatar.vue

```diff
@@ -1,14 +1,21 @@
 <template>
-  <avatar
-    class="oc-avatar"
-    :username="userName"
-    :size="width"
-    :src="src"
-    :aria-label="accessibleLabel === '' ? null : accessibleLabel"
-    :aria-hidden="accessibleLabel === '' ? 'true' : null"
-    :focusable="accessibleLabel === '' ? 'false' : null"
-    :role="accessibleLabel === '' ? null : 'img'"
-  />
+  <div class="oc-avatar-wrapper">
```

Comment:
This is not valid in terms of a11y/HTML validation, since we're using the avatar component inside a button component (e.g. user menu)